server/jobs/index.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+import fs from 'fs';
+
+const getJobs = (folderName) => {
```

Comment:
Is this not meant to have a docstring?

Reply:
Yes, it is meant to have a docstring. I would add it.